testsuite/api/test_omsExport.py

```diff
@@ -0,0 +1,65 @@
+## status: correct
+## teardown_command: rm -rf test_omsExport-lua/
+## linux: yes
+## linux32: yes
+## mingw: yes
+## win: yes
+## mac: yes
+
+from OMSimulator import OMSimulator
+oms = OMSimulator()
+
+oms.setCommandLineOption("--suppressPath=true")
+oms.setTempDirectory("./test_omsExport-lua/")
```

Comment:
This should probably be `oms.setTempDirectory("./test_omsExport-py/")` but does not really matter I guess.